Resources/public/js/views/tabs/ez-locationviewversionstabview.js

```diff
@@ -112,12 +115,12 @@ YUI.add('ez-locationviewversionstabview', function (Y) {
         },
 
         /**
-         * Enables the `buttons corresponding to the archived list.
+         * Enables the Archived buttons if the selection in the list is right
          *
-         * @method _enableArchivedVersionButtons
+         * @method _setArchivedButtonsState
          * @protected
          */
-        _enableArchivedVersionButtons: function () {
+        _setArchivedButtonsState: function () {
```

Comment:
It was not in the last release because BC has been broken in my previous PR when I renamed `_enableCreateDraftFromArchivedVersionButton` to `_enableArchivedVersionButtons`. I will fix the BC in this PR and add a depreciation message.